plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
```

Comment:
This is redundant information from the `.plug` file, can you check upstream to see a way around this? Maintaining two sets of docs is  hard.